In [157]:
# Veri Filtreleme & Sıralama Notebook
# Bu notebook .ipynb olarak çalıştırılmak üzere hazırlandı.
# İçinde hem kodlar hem de veriye dair analiz (yorum) hücreleri bulunuyor.

In [159]:
import pandas as pd

In [163]:
data = pd.read_csv('country.csv')
data.head()

Country                               Region  Population  \
0     Afghanistan         ASIA (EX. NEAR EAST)             31056997   
1         Albania   EASTERN EUROPE                          3581655   
2         Algeria   NORTHERN AFRICA                        32930091   
3  American Samoa   OCEANIA                                   57794   
4         Andorra   WESTERN EUROPE                            71201   

   Area (sq. mi.) Pop. Density (per sq. mi.) Coastline (coast/area ratio)  \
0          647500                       48,0                         0,00   
1           28748                      124,6                         1,26   
2         2381740                       13,8                         0,04   
3             199                      290,4                        58,29   
4             468                      152,1                         0,00   

  Net migration Infant mortality (per 1000 births)  GDP ($ per capita)  \
0         23,06                             163,07               700.0   
1         -4,93                              21,52              4500.0   
2         -0,39                                 31              6000.0   
3        -20,71                               9,27              8000.0   
4           6,6                               4,05             19000.0   

  Literacy (%) Phones (per 1000) Arable (%) Crops (%) Other (%) Climate  \
0         36,0               3,2      12,13      0,22     87,65       1   
1         86,5              71,2      21,09      4,42     74,49       3   
2         70,0              78,1       3,22      0,25     96,53       1   
3         97,0             259,5         10        15        75       2   
4        100,0             497,2       2,22         0     97,78       3   

  Birthrate Deathrate Agriculture Industry Service  
0      46,6     20,34        0,38     0,24    0,38  
1     15,11      5,22       0,232    0,188   0,579  
2     17,14      4,61       0,101      0,6   0,298  
3     22,46      3,27         NaN      NaN     NaN  
4      8,71      6,25         NaN      NaN     NaN

In [165]:
# Veri Seti Açıklaması:
# - Country: Ülkenin adı
# - Region: Ülkenin bulunduğu bölge (ör. ASIA, EASTERN EUROPE, NORTHERN AFRICA vb.)
# - Population: Ülkenin toplam nüfusu
# - Area (sq. mi.): Ülkenin yüzölçümü (mil kare olarak)
# - Pop. Density (per sq. mi.): Nüfus yoğunluğu (mil kare başına düşen kişi sayısı)
# - Coastline (coast/area ratio): Sahil uzunluğunun, ülkenin toplam alanına oranı
# - Net migration: Net göç oranı (ülkeye giren veya çıkan kişi sayısına göre)
# - Infant mortality (per 1000 births): Bebek ölüm oranı (1000 doğum başına)
# - GDP ($ per capita): Kişi başına düşen Gayri Safi Yurtiçi Hasıla (GSYİH)
# - Literacy (%): Okur-yazarlık oranı
# - Phones (per 1000): Her 1000 kişi başına düşen telefon sayısı
# - Arable (%): Tarıma elverişli arazi yüzdesi
# - Crops (%): Ekilebilir ürünlerin yüzdesi
# - Other (%): Diğer arazi kullanımı yüzdesi
# - Climate: Ülkenin iklim kategorisi (numerik değer)
# - Birthrate: Doğum oranı
# - Deathrate: Ölüm oranı
# - Agriculture: Tarım sektörünün ekonomideki payı
# - Industry: Sanayi sektörünün ekonomideki payı
# - Service: Hizmet sektörünün ekonomideki payı

#### Nüfusa Göre Azalan Sıralama

In [168]:
data_sorted_population = data.sort_values(by='Population', ascending=False)
data_sorted_population.head(10)

Country                               Region  Population  \
42           China         ASIA (EX. NEAR EAST)           1313973713   
94           India         ASIA (EX. NEAR EAST)           1095351995   
214  United States   NORTHERN AMERICA                      298444215   
95       Indonesia         ASIA (EX. NEAR EAST)            245452739   
27          Brazil               LATIN AMER. & CARIB       188078227   
156       Pakistan         ASIA (EX. NEAR EAST)            165803560   
16      Bangladesh         ASIA (EX. NEAR EAST)            147365352   
169         Russia                  C.W. OF IND. STATES    142893540   
152        Nigeria   SUB-SAHARAN AFRICA                    131859731   
103          Japan         ASIA (EX. NEAR EAST)            127463611   

     Area (sq. mi.) Pop. Density (per sq. mi.) Coastline (coast/area ratio)  \
42          9596960                      136,9                         0,15   
94          3287590                      333,2                         0,21   
214         9631420                       31,0                         0,21   
95          1919440                      127,9                         2,85   
27          8511965                       22,1                         0,09   
156          803940                      206,2                         0,13   
16           144000                     1023,4                         0,40   
169        17075200                        8,4                         0,22   
152          923768                      142,7                         0,09   
103          377835                      337,4                         7,87   

    Net migration Infant mortality (per 1000 births)  GDP ($ per capita)  \
42           -0,4                              24,18              5000.0   
94          -0,07                              56,29              2900.0   
214          3,41                                6,5             37800.0   
95              0                               35,6              3200.0   
27          -0,03                              29,61              7600.0   
156         -2,77                              72,44              2100.0   
16          -0,71                               62,6              1900.0   
169          1,02                              15,39              8900.0   
152          0,26                               98,8               900.0   
103             0                               3,26             28200.0   

    Literacy (%) Phones (per 1000) Arable (%) Crops (%) Other (%) Climate  \
42          90,9             266,7       15,4      1,25     83,35     1,5   
94          59,5              45,4       54,4      2,74     42,86     2,5   
214         97,0             898,0      19,13      0,22     80,65       3   
95          87,9              52,0      11,32      7,23     81,45       2   
27          86,4             225,3       6,96       0,9     92,15       2   
156         45,7              31,8      27,87      0,87     71,26       1   
16          43,1               7,3      62,11      3,07     34,82       2   
169         99,6             280,6       7,33      0,11     92,56     NaN   
152         68,0               9,3      31,29      2,96     65,75     1,5   
103         99,0             461,2      12,19      0,96     86,85       3   

    Birthrate Deathrate Agriculture Industry Service  
42      13,25      6,97       0,125    0,473   0,403  
94      22,01      8,18       0,186    0,276   0,538  
214     14,14      8,26        0,01    0,204   0,787  
95      20,34      6,25       0,134    0,458   0,408  
27      16,56      6,17       0,084      0,4   0,516  
156     29,74      8,23       0,216    0,251   0,533  
16       29,8      8,27       0,199    0,198   0,603  
169      9,95     14,65       0,054    0,371   0,575  
152     40,43     16,94       0,269    0,487   0,244  
103      9,37      9,16       0,017    0,258   0,725

##### Gözlem:
- Nüfus açısından en kalabalık ülkeler Çin, Hindistan ve ABD olarak sıralanıyor. 
- Asya ülkeleri (Çin, Hindistan, Endonezya, Pakistan, Bangladeş, Japonya) listeye ağırlıklı olarak hakim.
- Nüfus yoğunluğu bakımından Bangladeş (1023,4 kişi/sq.mi.) ve Hindistan (333,2 kişi/sq.mi.) oldukça yüksek yoğunluğa sahip, buna karşılık Rusya (8,4 kişi/sq.mi.) ve ABD (31 kişi/sq.mi.) düşük yoğunlukta.
- GSYİH (GDP per capita) açısından ABD ve Japonya öne çıkarken, Bangladeş, Pakistan gibi ülkeler oldukça düşük değerlerde.
- Literacy (%) değerleri yüksek olan ülkeler Japonya, Rusya, ABD gibi ülkeler.
- Bu sıralama yalnızca nüfus büyüklüğüne odaklanıyor, diğer ekonomik veya sosyal göstergeleri hesaba katmıyor.

##### Yorum:
- Nüfus büyüklüğü, ülke ekonomisi, altyapı ve kaynak yönetimi açısından kritik bir faktör. 
- Yüksek nüfuslu ülkelerde altyapı ve sosyal hizmetler üzerindeki baskı artabilir; düşük nüfus yoğunluğu ise kaynakların daha geniş alana yayılmasını sağlar.
- Analiz, yalnızca nüfusa göre sıralandığından, GDP, Literacy veya Population Density gibi diğer göstergelerle birlikte incelendiğinde daha anlamlı çıkarımlar yapılabilir.

#### GDP per capita’ye göre artan sıralama

In [172]:
data_sorted_gdp = data.sort_values(by='GDP ($ per capita)', ascending=True)
data_sorted_gdp.head(10)

Country                               Region  Population  \
58             East Timor         ASIA (EX. NEAR EAST)              1062777   
188               Somalia   SUB-SAHARAN AFRICA                      8863338   
183          Sierra Leone   SUB-SAHARAN AFRICA                      6005250   
33                Burundi   SUB-SAHARAN AFRICA                      8090068   
200              Tanzania   SUB-SAHARAN AFRICA                     37445392   
74             Gaza Strip   NEAR EAST                               1428757   
125                Malawi   SUB-SAHARAN AFRICA                     13013926   
0             Afghanistan         ASIA (EX. NEAR EAST)             31056997   
46   Congo, Repub. of the   SUB-SAHARAN AFRICA                      3702314   
63                Eritrea   SUB-SAHARAN AFRICA                      4786994   

     Area (sq. mi.) Pop. Density (per sq. mi.) Coastline (coast/area ratio)  \
58            15007                       70,8                         4,70   
188          637657                       13,9                         0,47   
183           71740                       83,7                         0,56   
33            27830                      290,7                         0,00   
200          945087                       39,6                         0,15   
74              360                     3968,8                        11,11   
125          118480                      109,8                         0,00   
0            647500                       48,0                         0,00   
46           342000                       10,8                         0,05   
63           121320                       39,5                         1,84   

    Net migration Infant mortality (per 1000 births)  GDP ($ per capita)  \
58              0                              47,41               500.0   
188          5,37                              116,7               500.0   
183             0                             143,64               500.0   
33          -0,06                              69,29               600.0   
200         -2,06                              98,54               600.0   
74            1,6                              22,93               600.0   
125             0                             103,32               600.0   
0           23,06                             163,07               700.0   
46          -0,17                              93,86               700.0   
63              0                              74,87               700.0   

    Literacy (%) Phones (per 1000) Arable (%) Crops (%) Other (%) Climate  \
58          58,6               NaN       4,71      0,67     94,62       2   
188         37,8              11,3       1,67      0,04     98,29       1   
183         31,4               4,0       6,98      0,89     92,13       2   
33          51,6               3,4      35,05     14,02     50,93       2   
200         78,2               4,0       4,52      1,08      94,4     NaN   
74           NaN             244,3      28,95     21,05        50       3   
125         62,7               7,9      23,38      1,49     75,13       2   
0           36,0               3,2      12,13      0,22     87,65       1   
46          83,8               3,7       0,51      0,13     99,36       2   
63          58,6               7,9       4,95      0,03     95,02     1,5   

    Birthrate Deathrate Agriculture Industry Service  
58      26,99      6,24       0,085    0,231   0,684  
188     45,13     16,63        0,65      0,1    0,25  
183     45,76     23,03        0,49     0,31    0,21  
33      42,22     13,46       0,463    0,203   0,334  
200     37,71     16,39       0,432    0,172   0,396  
74      39,45       3,8        0,03    0,283   0,687  
125     43,13     19,33       0,342    0,158   0,499  
0        46,6     20,34        0,38     0,24    0,38  
46      42,57     12,93       0,062     0,57   0,369  
63      34,33       9,6       0,102    0,254

##### Gözlem:
- GDP ($ per capita) açısından en düşük değerler 500-700 USD civarında. Bu ülkeler genellikle Afrika (Somalia, Sierra Leone, Malawi, Eritrea) ve Asya (East Timor, Afghanistan) bölgelerinde yer alıyor.
- En düşük GSYİH’lı ülkelerde literacy (%), infant mortality ve population yoğunluğu farklılık gösteriyor. Örneğin, literacy oranı 31,4 (Sierra Leone) ile 62,7 (Malawi) arasında değişiyor.
- Phones per 1000 ve diğer altyapı göstergeleri düşük GSYİH’lı ülkelerde oldukça sınırlı. Örneğin East Timor’da telefon sayısı veri olarak eksik, Gaza Strip’te 244,3 ile yüksek bir istisna.
- Nüfus yoğunluğu bazı düşük GSYİH’lı ülkelerde yüksek (Burundi: 290,7 kişi/sq.mi, Gaza Strip: 3968,8 kişi/sq.mi), bazı ülkelerde düşük (Somalia: 13,9 kişi/sq.mi).

##### Yorum:
- Kişi başına düşen GSYİH, ülkelerin ekonomik gelişmişlik düzeyini yansıtır ve altyapı, eğitim, sağlık gibi sosyal göstergelerle yakından ilişkilidir.
- Bu sıralama, ekonomik açıdan en zayıf ülkeleri hızlıca belirlemek için kullanılabilir.
- Bazı ülkelerde nüfus yoğunluğu ve altyapı göstergeleri yüksekken GSYİH düşük; bu, kaynakların etkin kullanımında ve gelir dağılımında problemler olabileceğine işaret ediyor.
- Analiz, düşük GSYİH’lı ülkelerde sosyal ve ekonomik politikaların iyileştirilmesi gerektiğini ortaya koyuyor.

#### Population > 10 milyon filtreleme

In [176]:
population_over_10m = data[data['Population'] > 10_000_000]
population_over_10m

Country                               Region  Population  \
0    Afghanistan         ASIA (EX. NEAR EAST)             31056997   
2        Algeria   NORTHERN AFRICA                        32930091   
5         Angola   SUB-SAHARAN AFRICA                     12127071   
8      Argentina               LATIN AMER. & CARIB        39921833   
11     Australia   OCEANIA                                20264082   
..            ...                                  ...         ...   
218    Venezuela               LATIN AMER. & CARIB        25730435   
219      Vietnam         ASIA (EX. NEAR EAST)             84402966   
224        Yemen   NEAR EAST                              21456188   
225       Zambia   SUB-SAHARAN AFRICA                     11502010   
226     Zimbabwe   SUB-SAHARAN AFRICA                     12236805   

     Area (sq. mi.) Pop. Density (per sq. mi.) Coastline (coast/area ratio)  \
0            647500                       48,0                         0,00   
2           2381740                       13,8                         0,04   
5           1246700                        9,7                         0,13   
8           2766890                       14,4                         0,18   
11          7686850                        2,6                         0,34   
..              ...                        ...                          ...   
218          912050                       28,2                         0,31   
219          329560                      256,1                         1,05   
224          527970                       40,6                         0,36   
225          752614                       15,3                         0,00   
226          390580                       31,3                         0,00   

    Net migration Infant mortality (per 1000 births)  GDP ($ per capita)  \
0           23,06                             163,07               700.0   
2           -0,39                                 31              6000.0   
5               0                             191,19              1900.0   
8            0,61                              15,18             11200.0   
11           3,98                               4,69             29000.0   
..            ...                                ...                 ...   
218         -0,04                               22,2              4800.0   
219         -0,45                              25,95              2500.0   
224             0                               61,5               800.0   
225             0                              88,29               800.0   
226             0                              67,69              1900.0   

    Literacy (%) Phones (per 1000) Arable (%) Crops (%) Other (%) Climate  \
0           36,0               3,2      12,13      0,22     87,65       1   
2           70,0              78,1       3,22      0,25     96,53       1   
5           42,0               7,8       2,41      0,24     97,35     NaN   
8           97,1             220,4      12,31      0,48     87,21       3   
11         100,0             565,5       6,55      0,04     93,41       1   
..           ...               ...        ...       ...       ...     ...   
218         93,4             140,1       2,95      0,92     96,13       2   
219         90,3             187,7      19,97      5,95     74,08       2   
224         50,2              37,2       2,78      0,24     96,98       1   
225         80,6               8,2       7,08      0,03      92,9       2   
226         90,7              26,8       8,32      0,34     91,34       2   

    Birthrate Deathrate Agriculture Industry Service  
0        46,6     20,34        0,38     0,24    0,38  
2       17,14      4,61       0,101      0,6   0,298  
5       45,11      24,2       0,096    0,658   0,246  
8       16,73      7,55       0,095    0,358   0,547  
11      12,14      7,51       0,038    0,262     0,7  
..        ...       ...         ...      ...     ...

##### Gözlem:
* Filtreleme sonucunda, dünya genelinde 10 milyonun üzerinde nüfusa sahip ülkeler seçildi.
* Bu ülkeler Asya, Afrika, Latin Amerika ve Okyanusya gibi farklı bölgelerden geliyor.
* Nüfus bakımından en büyük ülkeler Çin, Hindistan, ABD ve Endonezya gibi ülkeler.
* Bu ülkelerde nüfus yoğunluğu, yüzölçümü ve diğer demografik göstergeler arasında büyük farklılıklar gözlemleniyor. Örneğin Çin ve Hindistan, yüzölçümü büyük olmasına rağmen yüksek nüfus yoğunluğuna sahipken, Avustralya çok düşük nüfus yoğunluğuna sahip.

##### Yorum:
* Nüfusu 10 milyonun üzerinde olan ülkeler, ekonomik, sosyal ve altyapısal planlama açısından daha büyük ölçekli politikalar gerektiriyor.
* Bu filtreleme, yüksek nüfuslu ülkelerin analizlerde ayrı bir gruplandırma veya karşılaştırma yapılabilmesini sağlıyor.
* Aynı zamanda, yüksek nüfuslu ülkelerde diğer göstergeler (GSYİH, okur-yazarlık oranı, sağlık göstergeleri) daha geniş varyans gösteriyor, bu da veri analizinde dikkat edilmesi gereken bir noktadır.

#### En yüksek okur-yazarlık oranına sahip ilk 5 ülke

In [180]:
top_5_literacy = data.sort_values(by='Literacy (%)', ascending=False).head(5)
top_5_literacy

Country                               Region  Population  \
53   Czech Republic   EASTERN EUROPE                         10235455   
93          Iceland   WESTERN EUROPE                           299388   
163          Poland   EASTERN EUROPE                         38536869   
114          Latvia   BALTICS                                 2274735   
64          Estonia   BALTICS                                 1324333   

     Area (sq. mi.) Pop. Density (per sq. mi.) Coastline (coast/area ratio)  \
53            78866                      129,8                         0,00   
93           103000                        2,9                         4,83   
163          312685                      123,3                         0,16   
114           64589                       35,2                         0,82   
64            45226                       29,3                         8,39   

    Net migration Infant mortality (per 1000 births)  GDP ($ per capita)  \
53           0,97                               3,93             15700.0   
93           2,38                               3,31             30900.0   
163         -0,49                               8,51             11100.0   
114         -2,23                               9,55             10200.0   
64          -3,16                               7,87             12300.0   

    Literacy (%) Phones (per 1000) Arable (%) Crops (%) Other (%) Climate  \
53          99,9             314,3       39,8      3,05     57,15       3   
93          99,9             647,7       0,07         0     99,93       3   
163         99,8             306,3      45,91      1,12     52,97       3   
114         99,8             321,4      29,67      0,47     69,86       3   
64          99,8             333,8      16,04      0,45     83,51       3   

    Birthrate Deathrate Agriculture Industry Service  
53       9,02     10,59       0,034    0,393   0,573  
93      13,64      6,72       0,086     0,15   0,765  
163      9,85      9,89        0,05    0,311    0,64  
114      9,24     13,66        0,04    0,261   0,699  
64      10,04     13,25        0,04    0,294   0,666

##### Gözlem:
* Literacy (%) sütununa göre sıralama yapıldığında, en yüksek okur-yazarlık oranına sahip ülkeler çoğunlukla Avrupa bölgesinden (Eastern Europe, Western Europe, Baltics) geliyor.
* İlk 5 ülkenin okur-yazarlık oranları %99,8–%99,9 aralığında, neredeyse tam okur-yazarlık seviyesinde.
* Bu ülkeler arasında nüfus yoğunluğu ve yüzölçümü farklılıkları göze çarpıyor; örneğin, Polonya büyük bir nüfusa sahipken, Estonya ve Latvia nispeten küçük nüfuslu ülkeler.
* Kişi başına düşen GSYİH (GDP per capita) da yüksek seviyelerde (10.200–30.900 USD), bu durum okur-yazarlık oranı ile ekonomik refah arasında bir korelasyon olabileceğini düşündürüyor.

##### Yorum:
* Yüksek okur-yazarlık oranına sahip ülkeler, genellikle eğitim ve sosyal altyapıya güçlü yatırım yapan ülkeler olarak öne çıkıyor.
* Bu ülkelerde ekonomik göstergeler ve yaşam kalitesi de genellikle yüksek; dolayısıyla eğitim düzeyi, ülke kalkınması ve ekonomik refah için önemli bir belirleyici olarak yorumlanabilir.
* Analiz, okur-yazarlık oranı yüksek ülkelerin küçük veya büyük nüfuslu olabilmesine rağmen, sosyal ve ekonomik göstergelerde avantajlı olduklarını ortaya koyuyor.

#### En yüksek nüfus yoğunluğuna sahip ilk 10 ülke

In [184]:
top_10_density = data.sort_values(by='Pop. Density (per sq. mi.)', ascending=False).head(10)
top_10_density

Country                               Region  Population  \
9       Armenia                  C.W. OF IND. STATES      2976372   
186    Slovenia   EASTERN EUROPE                          2010347   
202        Togo   SUB-SAHARAN AFRICA                      5548702   
12      Austria   WESTERN EUROPE                          8192880   
168     Romania   EASTERN EUROPE                         22303552   
77        Ghana   SUB-SAHARAN AFRICA                     22409572   
140  Montserrat               LATIN AMER. & CARIB            9439   
13   Azerbaijan                  C.W. OF IND. STATES      7961619   
56     Dominica               LATIN AMER. & CARIB           68910   
206      Turkey   NEAR EAST                              70413958   

     Area (sq. mi.) Pop. Density (per sq. mi.) Coastline (coast/area ratio)  \
9             29800                       99,9                         0,00   
186           20273                       99,2                         0,23   
202           56785                       97,7                         0,10   
12            83870                       97,7                         0,00   
168          237500                       93,9                         0,09   
77           239460                       93,6                         0,23   
140             102                       92,5                        39,22   
13            86600                       91,9                         0,00   
56              754                       91,4                        19,63   
206          780580                       90,2                         0,92   

    Net migration Infant mortality (per 1000 births)  GDP ($ per capita)  \
9           -6,47                              23,28              3500.0   
186          1,12                               4,45             19000.0   
202             0                              66,61              1500.0   
12              2                               4,66             30000.0   
168         -0,13                              26,43              7000.0   
77          -0,64                              51,43              2200.0   
140             0                               7,35              3400.0   
13           -4,9                              81,74              3400.0   
56         -13,87                              14,15              5400.0   
206             0                              41,04              6700.0   

    Literacy (%) Phones (per 1000) Arable (%) Crops (%) Other (%) Climate  \
9           98,6             195,7      17,55       2,3     80,15       4   
186         99,7             406,1        8,6      1,49     89,91     NaN   
202         60,9              10,6      46,15      2,21     51,64       2   
12          98,0             452,2      16,91      0,86     82,23       3   
168         98,4             196,9      40,82      2,25     56,93       3   
77          74,8              14,4      16,26      9,67     74,07       2   
140         97,0               NaN         20         0        80       2   
13          97,0             137,1      19,63      2,71     77,66       1   
56          94,0             304,8       6,67        20     73,33       2   
206         86,5             269,5      30,93      3,31     65,76       3   

    Birthrate Deathrate Agriculture Industry Service  
9       12,07      8,23       0,239    0,343   0,418  
186      8,98     10,31       0,028    0,369   0,603  
202     37,01      9,83       0,395    0,204   0,401  
12       8,74      9,76       0,018    0,304   0,678  
168      10,7     11,77       0,101     0,35   0,549  
77      30,52      9,72       0,366    0,246   0,387  
140     17,59       7,1         NaN      NaN     NaN  
13      20,74      9,75       0,141    0,457   0,402  
56      15,27      6,73       0,177    0,328   0,495  
206     16,62      5,97       0,117    0,298   0,585

##### Gözlem:
* Sıralamada okur-yazarlık oranı yüksek olan ülkeler arasında Avrupa ülkeleri (Slovenia, Austria, Romania), Orta Doğu ve Yakın Doğu ülkeleri (Turkey, Armenia, Azerbaijan) ve bazı Afrika ülkeleri (Ghana, Togo) yer alıyor.
* Literacy (%) oranları çoğunlukla %90’ın üzerinde; en yüksek değer 99,7 (Slovenia), en düşük yüksek oran ise 86,5 (Turkey).
* Nüfus açısından büyük farklılıklar mevcut: örneğin Turkey 70 milyonu aşan bir nüfusa sahipken, Montserrat ve Dominica gibi bazı ülkelerin nüfusu çok küçük (10 bin civarı).
* Kişi başına düşen GSYİH’de de farklılıklar var: Austria 30.000 USD civarında yüksek gelir sağlarken, Togo ve Ghana 2.000–2.200 USD civarında düşük gelir seviyelerinde.

##### Yorum:
* Yüksek okur-yazarlık oranı her zaman ekonomik refah ile paralel değil; örneğin, Ghana ve Togo’da okur-yazarlık yüksek olmasına rağmen GSYİH düşük.
* Avrupa ülkeleri hem yüksek okur-yazarlık hem de yüksek kişi başı gelir ile öne çıkıyor.
* Türkiye, okur-yazarlık oranı %86,5 ile yüksek olmakla birlikte, GSYİH ve nüfus açısından orta seviyelerde konumlanıyor.
* Bu analiz, okur-yazarlık oranının ekonomik ve demografik göstergelerle tek başına değerlendirilemeyeceğini, bölgesel ve sosyal faktörlerin de göz önünde bulundurulması gerektiğini gösteriyor.

## Genel Analiz

### 1. Nüfus:
* En kalabalık ülkeler Çin (1,31 milyar), Hindistan (1,09 milyar) ve ABD (298 milyon) olarak öne çıkıyor.
* Nüfus yoğunluğu açısından Bangladeş (1023 kişi/mil²) ve Gaza Strip (3968 kişi/mil²) gibi bölgeler çok yoğun, Rusya (8,4 kişi/mil²) ve Avustralya (2,6 kişi/mil²) gibi ülkelerse seyrek nüfuslu.

### 2. Ekonomi (GDP per capita):
* Kişi başına düşen gelir en yüksek olan ülkeler arasında Avrupa ülkeleri (Austria, Slovenia, Iceland) ve ABD bulunuyor.
* Düşük gelirli ülkeler çoğunlukla Afrika ve Asya’daki gelişmekte olan ülkeler (Togo, Ghana, East Timor).
* Ekonomik farklılıklar, yaşam standardı ve altyapı göstergeleri ile paralellik gösteriyor.

### 3. Okur-Yazarlık Oranı (Literacy %):
* En yüksek okur-yazarlık oranları %99 civarında ve çoğunlukla Avrupa ülkelerinde.
* Afrika ve bazı Asya ülkelerinde okur-yazarlık daha düşük, ancak yüksek okur-yazarlık ile yüksek gelir her zaman paralel değil (ör. Ghana, Togo).

### 4. Demografi ve Sağlık:
* Bebek ölüm oranları düşük gelirli ülkelerde yüksek (Somalia 116,7/1000), yüksek gelirli ülkelerde düşük (Iceland 3,31/1000).
* Doğum ve ölüm oranları bölgesel farklılıklar gösteriyor; yüksek doğum oranı genellikle düşük gelirli ülkelerde.

### 5. Arazi ve Tarım:
* Tarıma elverişli alan ve ekilebilir ürün oranı ülkeden ülkeye çok değişiyor.
* Orta Doğu ve Afrika’da tarıma elverişli arazi düşük, Avrupa’da ve bazı Asya ülkelerinde daha yüksek.

#### Genel Yorum:
Veri seti, ülkelerin nüfus, ekonomik, sağlık, eğitim ve coğrafi göstergelerini kapsayarak kapsamlı bir karşılaştırma imkânı sağlıyor.
Gözlemler, gelir, eğitim ve sağlık göstergeleri arasında güçlü bir bölgesel farklılık olduğunu gösteriyor.
Nüfus yoğunluğu, ekonomik refah ve okur-yazarlık gibi göstergeler, ülkelerin kalkınma seviyesini ve altyapı ihtiyaçlarını anlamak için kullanılabilir.